presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix ``IS DISTINCT FROM`` for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve performance of :func:`array_join`.
+* Improve efficiency of :func:`ST_Buffer`. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Improve efficiency by using JTS instead of Esri for many geometrical operations and fix incorrect calculation of extreme points in certain cases (:issue:`14031`).
+* Add forms of :func:`approx_percentile` accepting an accuracy parameter.
+* Add a new session property ``aggregation_partitioning_merging_strategy`` to configure partition merging strategy when adding exchange around aggregation node.
+* Add ``IGNORE NULLS`` clause to various :doc:`/functions/window`.
+
+Hive Changes
+____________
+* Improve string column stats by including the sum of all strings lengths when the individual min/max of the string is too long in ORC writer.
```

Comment:
Should it be "all string lengths" or "all strings lengths"? I don't know what's the right grammar. 😂

Reply:
either `all strings' lengths` (with an apostrophe after the s) or 'all string lengths`  as rongrong suggested.

Reply:
changed it to "all strings' lengths"